Documentation/spelling_wordlist.txt

```diff
@@ -245,6 +246,7 @@ committer
 committers
 conf
 config
+configmap
```

Comment:
nit: I suspect that if you use `ConfigMap` in single back-ticks rather than plain then you won't need this.

Reply:
Aye, you are right;-)